patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -156,20 +156,20 @@
 +         return isBeach && hasWater;
 +      }
 +      return false;
-+  }
++   }
 +
-+  @Override
-+  public final java.util.Set<net.minecraft.util.ResourceLocation> getTags() {
-+     return reverseTags.getTagNames();
-+  }
++   @Override
++   public final java.util.Set<net.minecraft.util.ResourceLocation> getTags() {
```

Comment:
the heck is going on here?

Reply:
Adjustments to the "tab" spacing to make it equal to others in this file. (3 spaces per "tab". It might not look like it, but it is.)